### ISSUE TITLE
Browser settings page - Use system fonts for settings items

### ISSUE THREAD

**Issue by aguscruiz** [brave/brave-browser#41559]
- Moving from Poppins to use system fonts in the settings pages.
- Misc small changes to styling of items

**Comment by aguscruiz** [comment]
https://github.com/brave/brave-core/pull/25937

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.74.8 Chromium: 131.0.6778.69 (Official Build) beta (x86_64)
-- | --
Revision | 0e56a6339728b1f1569eb3bdbe8261f2d5785d3b
OS | macOS Version 14.7.1 (Build 23H222)
```

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.74.8 Chromium: 131.0.6778.69 (Official Build) beta (64-bit)
-- | --
Revision | 0e56a6339728b1f1569eb3bdbe8261f2d5785d3b
OS | Windows 11 Version 23H2 (Build 22631.4460)
```


brave://settings/braveContent

1.74.8 | 1.73.89 | 131.0.6778.70 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/0cd604a1-105c-40db-82ee-9a87f1d2df50)|![Image](https://github.com/user-attachments/assets/37aa4925-dc46-48b3-b910-36a3d1691912)|![Image](https://github.com/user-attachments/assets/40543480-c725-419f-a86f-f114bd1bdf3b)

1.74.8 | 1.73.89 | 131.0.6778.70 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/93773ba0-6167-41fa-be95-abc1cb4e73f3)|![Image](https://github.com/user-attachments/assets/747d4a84-7b50-47a7-9ef6-233045e97815)|![Image](https://github.com/user-attachments/assets/750e7cc2-1196-4b36-baa3-a1e0654acf44)

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave	1.74.19 Chromium: 131.0.6778.108 (Official Build) beta (64-bit) 
Revision	12080da255dfc6f1f55b78bd96cd2f18bce5cb8a
OS	Linux
````



Verified test plan from https://github.com/brave/brave-core/pull/25937#issue-2579827657:

<details>
<summary>Case 1 - Font should match the System font on the chrome://settings page - Pending</summary>

Compared font on `1.73.97 ` (current released version) to `1.74.19` and Chrome `131.0.6778.86` for various settings.
Confirmed while `1.73.89` uses Poppins for setting font, `1.74.8` and Chrome `131.0.6778.108` do not, they use the same system font.

Note - didn't check each setting font text, spot checked a few and put some screenshots below.

**_brave://settings/braveContent_**

1.74.19 | 1.73.97 | 131.0.6778.108 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/6830e8cd-6297-491d-a39c-b1276fa10eb5)|![Image](https://github.com/user-attachments/assets/9cb8abdf-c4dc-4186-987b-b550a7f204af)|![Image](https://github.com/user-attachments/assets/6408596a-ad0c-4c79-a899-b6a2f999a6cc)


**_brave://settings/system_**

1.74.19 | 1.73.97 | 131.0.6778.108 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/bddf0a1a-f90b-4344-bfa2-97755134d661)|![Image](https://github.com/user-attachments/assets/a1cb9555-d25f-486b-8619-ae198a28c887)|![Image](https://github.com/user-attachments/assets/b3046b50-4c51-42e2-b728-3f28450ca356)


**_brave://settings/autofill_**

1.74.19 | 1.73.97 | 131.0.6778.108 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/e90786cf-7194-4312-b9d6-3bcde969fa13)|![Image](https://github.com/user-attachments/assets/4061eae6-9457-4af4-87cf-99627062c23f)|![Image](https://github.com/user-attachments/assets/02e6d6b3-c2de-404e-80cc-daf722329cf3)

**_brave://settings/help_**

1.74.19 | 1.73.97 | 131.0.6778.108 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/5a859b2c-fb60-443d-a5c9-e3b361f96b68)|![Image](https://github.com/user-attachments/assets/7e1de343-aaaa-4c91-af3d-2667dccd7629)|![Image](https://github.com/user-attachments/assets/a7d0c889-3653-4914-a89d-ca12725032ba)
</details>

<details>
<summary>Case 2 - Font should match the system font on the left navigation of the settings - Pending</summary>

Compared font on `1.73.97` (current released version) to `1.74.19` and Chrome `131.0.6778.108` for various items in left navigation bar.
Confirmed while `1.73.97` uses Poppins for setting font, `1.74.19` and Chrome `131.0.6778.108` do not, they use the same system font.

`Reset settings`

1.74.19 | 1.73.97 | 131.0.6778.108 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/fb25f6dd-297e-4e56-9cb3-c383719dc0de)|![Image](https://github.com/user-attachments/assets/80ff9818-52a8-438e-82bc-0aa0a68f3e31)|![Image](https://github.com/user-attachments/assets/5c5a28f2-6051-4aa8-8a7c-0dd87d08ef18)


`Privacy and Security`

1.74.19 | 1.73.97 | 131.0.6778.108 (chrome)
-------- | ---------- | ----------
![Image](https://github.com/user-attachments/assets/7ce47300-37f8-40e0-8b30-3440767e6fef)|![Image](https://github.com/user-attachments/assets/56f1b5a9-0aaf-47d5-9bdd-9e0ac1fff537)|![Image](https://github.com/user-attachments/assets/741aec4d-a33f-44a5-bfdc-f34d73aa5581)
</details>

<details>
<summary>Case 3 - Poppins should still be the font used for section titles in the settings pages - Pending</summary>

Compared font on `1.73.97` (current released version) to `1.74.19` for various section titles in the settings pages.
Confirmed `1.74.19` still uses Poppins.

`1.73.97` | `1.74.19`

Tor windows | Wallet | Leo
----- | ----- | -----
![Image](https://github.com/user-attachments/assets/5ddef822-1777-4e07-b331-840c0749add8)|![Image](https://github.com/user-attachments/assets/9b8fd58c-cf54-45dc-948f-5a95046228bd)|![Image](https://github.com/user-attachments/assets/b54be77c-d033-4623-aa4b-fae019916f61)
![Image](https://github.com/user-attachments/assets/71ba3ff9-6408-4ec8-a048-e792879e58ab)|![Image](https://github.com/user-attachments/assets/ba719271-a499-4461-ab2f-da1ae47b5709)|![Image](https://github.com/user-attachments/assets/3c402e89-13bc-43b1-a6c5-58b56d975ed7)

</details>
